Sources/ImageProcessor.swift

```diff
@@ -172,20 +173,38 @@ public struct ResizingImageProcessor: ImageProcessor {
     /// Target size of output image should be.
     public let targetSize: CGSize
     
+    
+    public enum ContentMode {
+        case none // This will be the default one
+        case aspectFit
+        case aspectFill
+    }
+    /// Target content mode of output image should be.
+    public let contentMode: UIViewContentMode
```

Comment:
This will change `ImageProcessor` to be UIKit only. We may want to keep it working on macOS too. So I suggest to not introducing `UIViewContentMode` here. We need to do change the `resize(to size: CGSize, for contentMode: UIViewContentMode)` implantation a bit too.